src/rf/js/src/home/views.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var React = require('react'),
+    asset = require('../core/utils').asset;
+
+var LoginView = React.createBackboneClass({
+    render: function() {
+        return <div>
```

Comment:
I'll update this in the next PR since this got moved to a separate file and I don't want to create a merge conflict.